src/lib/Controls.scss

```diff
@@ -180,3 +180,26 @@
         margin-left: 4px;
     }
 }
+
+.bp-annotations-group {
```

Comment:
Let's move these styles to their own stylesheet file and write them in the SUIT convention.

Reply:
> Let's move these styles to their own stylesheet file and write them in the SUIT convention.

Will do